src/lib/hostapd.rb

```diff
@@ -8,6 +8,9 @@
 require 'logger'
 require 'contracts'
 
+# RADAR: because otherwise ruby uses "Encoding:US-ASCII"
```

Comment:
reword: "because otherwise our docker ruby image mysteriously uses..."

Otherwise one day one might think this is some sane default ruby ships with by default :D